src/lib/util/arrays.ts

```diff
@@ -12,3 +15,19 @@ function chunk<T>(array: T[], size: number): T[][] {
 function chunkString(str: string, size: number): string[] {
   return chunk([...str], size).map((c) => c.join(''));
 }
+
+function zip<T, S>(a: T[], b: S[]) {
+  assert(
+    a.length <= b.length,
+    'zip(): first array must be at least as long as the second array'
```

Comment:
message is not consistent with assertion.

Reply:
🤦🏻